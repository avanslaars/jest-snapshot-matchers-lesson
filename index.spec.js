const { createPerson } = require('./index')

describe('createPerson', () => {
  it('Creates a person object', () => {
    const result = createPerson('John', 'Smith')
    expect(result).toMatchSnapshot()
  })
})
