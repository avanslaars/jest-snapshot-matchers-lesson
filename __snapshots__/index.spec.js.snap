// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createPerson Creates a person object 1`] = `
Object {
  "createdAt": Any<Date>,
  "firstName": "John",
  "id": Any<String>,
  "lastName": "Smith",
}
`;
